service/controller/appcatalog/v1/key/key.go

```diff
@@ -0,0 +1,37 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	versionBundleAnnotation = "giantswarm.io/version-bundle"
+)
+
```

Comment:
Lets add some tests for these. It's good for the `key` package to have coverage for when we need to change the CRDs.